.circleci/config.yml

```diff
@@ -28,13 +28,11 @@ executors:
     environment:
       GRADLE_OPTS: -Dorg.gradle.daemon=false -Dorg.gradle.parallel=true -Dorg.gradle.workers.max=4
       
-  quorum_ats_executor_xl: # 8cpu, 16G ram
-    docker:
-      - image: cimg/openjdk:11.0
-    resource_class: xlarge
+  quorum_ats_executor_med:
+    machine:
+      image: ubuntu-2004:202101-01
```

Comment:
The image here is changed from openjdk to ubuntu, is this intended?

Reply:
yes. updated the PR desc with extra info